pkg/controller/v1alpha2/core/scopes/healthscope/healthscope_controller_test.go

```diff
@@ -253,21 +249,6 @@ var _ = Describe("Test GetScopeHealthStatus", func() {
 					return nil
 				},
 			},
-			{
-				caseName:       "2 general workloads but one is unhealthy",
```

Comment:
why delete this case？

Reply:
because the general check logic (check any workloads having `status.readyReplicas`) is removed, so this case doesn't make sense anymore.